docs/configuration.md

```diff
@@ -46,6 +46,47 @@ are only shown to users handling the application, and super-users like
 the owner and reporter. In praticular, application members cannot see
 the additional user properties of each other.
 
+#### OIDC Hostname issues
```

Comment:
Since this is a rather long and very specific tip compared to the other general instructions, perhaps this could be moved to its own file or perhaps even posting this resolution as an issue (that you can close) and then creating the link here to that issue. 

I personally search first from the project issues when I have bump into problems. But I think it would be valuable to provide some hints on how to use various IdPs with REMS. A link from here would sound good. We don't have any such section yet but perhaps should.

What do you think @opqdonut ?

Reply:
hmm I'm not sure... my first reaction would be to put stuff like this in, say, `docs/contrib/keycloak.md` or something like that

Reply:
Yeah especially if we put some further instructions. And link from here to those.

Reply:
@katpavlova How do you feel about extracting this to `keycloack.md` and writing the rest of the how to use it? It can be short and up to the point. Link to the documentation of Keycloack for the necessary parts.

Reply:
Sure, that works for me. :)
What else would you like in the keycloak guide? Sort of a Quick Start for setting up a Keycloak IdP for REMS, followed by this documentation for hostname issues? Anything else?

Reply:
A quick start with Keycloak as how you set it up would be just great. Other developers that are experimenting would find it useful. I think we can provide one with e.g. Auth0 ourselves.

Reply:
Done, how is this?

Reply:
@Macroz I did as suggested, how is this?